traveler-swift-core/TravelerKit/TravelerRoute.swift

```diff
@@ -66,33 +66,29 @@ extension PassengerRoute: Route {
     }
 
     func transform(error: Error) -> Error {
-        guard case NetworkError.clientError(_, .some(let data)) = error else {
+        guard case NetworkError.clientError(400, .some(let data)) = error else {
             return error
         }
 
         guard let errorJSON = try? JSONSerialization.jsonObject(with: data, options: .allowFragments) as? [String: Any],
-           let errorCode = errorJSON["errorCode"] as? Int else {
+            let errorCode = errorJSON["errorCode"] as? Int else {
             Log("Bad JSON", data: String(data: data, encoding: .utf8), level: .error)
             return error
         }
 
-        switch (errorCode, errorJSON["errorData"]) {
-        case (2006, _):
+        switch errorCode {
+        case 2006:
             return BookingError.noPasses
-        case (2007, _):
+        case 2007:
             return BookingError.veryOldTraveler
-        case (2012-2014, _):
+        case 2012-2014:
```

Comment:
`2012..<2014`

Reply:
Wrong mapping, case below should have been 2015. Range 2012-2014 is correct.